libs/ui-rnative/src/lib/Components/Button/BaseButton.tsx

```diff
@@ -136,14 +152,14 @@ export const BaseButton = React.forwardRef<
         {loading && (
           <Spinner
             size={calculatedIconSize}
-            className='shrink-0 animate-spin text-inherit'
+            className={`shrink-0 animate-spin`}
```

Comment:
missing the new `getIconColorClass` here no ?

```suggestion
            className={cn("shrink-0 animate-spin", getIconColorClass())}
```

Reply:
ah maybe not, the issue with spinner is different then this one, then maybe just drop the brackets 
```suggestion
            className='shrink-0 animate-spin'
```